pkg/oc/admin/migrate/scc/scc.go

```diff
@@ -0,0 +1,240 @@
+package scc
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	extensions "k8s.io/api/extensions/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+	rbacv1helper "k8s.io/kubernetes/pkg/apis/rbac/v1"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/oc/cli/util/clientcmd"
+	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	securitytypedclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+
+	"github.com/spf13/cobra"
+)
+
+// pspPriorityAnnotationKey is a name of the annotation that could be present on PodSecurityPolicy and specifies its priority.
+// It should be a number. An empty value means 0 priority. Higher values means that the policy has higher priority.
+// This is an equivalent of the SCC.Priority field but applies to PSP and it's required for seamless transition.
+const pspPriorityAnnotationKey = "security.openshift.io/psp-priority"
+
+var (
+	internalMigrateSCCShort = "Converts SCCs to similar PSPs"
+	internalMigrateSCCLong  = templates.LongDesc(`
+		Converts SecurityContextConstraints to the equivalent PodSecurityPolicy objects.
+		For each SCC it also creates a ClusterRole with the same name and ClusterRoleBinding
+		for every user or group that SCC has.
+
+		NOTE: it does not modify anything by default.`)
```

Comment:
`It does not modify any input SCC.` or alike. You are actually modifying, but that's the whole point ;)

Reply:
Where I'm modifying? It only outputs the result to console.

Reply:
You're not at all? Why not? Why this is then different than any other migration command we have so far, in that case?